src/Text/Layout/Table/Primitives/Table.hs

```diff
@@ -10,32 +13,38 @@ import           Text.Layout.Table.Spec.Util
 
 
 -- | Draw a horizontal line that will use the delimiters around the
--- appropriately and visually separate by 'hSpace'.
+-- appropriately and visually separate by 'hSpace', or Nothing if
+-- all delimiters are null.
```

Comment:
* `null` -> `empty`
* Please use a separate sentence for describing the return value.
* `Nothing` -> `'Nothing'`